src/components/Input/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment } from 'react';
+
+import styles from './styles.module.scss';
+
+interface InputProps {
+  name: string;
+  inputRef?: any;
```

Comment:
type this, there is a way to type this with React `Ref<T>` type and you will have to research about what `T` generic type works in your case